syft/formats/syftjson/to_format_model_test.go

```diff
@@ -1,72 +1,184 @@
 package syftjson
 
 import (
+	"encoding/json"
 	"testing"
 
-	"github.com/scylladb/go-set/strset"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 
 	stereoscopeFile "github.com/anchore/stereoscope/pkg/file"
 	"github.com/anchore/syft/syft/file"
-	"github.com/anchore/syft/syft/formats/syftjson/model"
+	syftjson2 "github.com/anchore/syft/syft/formats/syftjson/model"
```

Comment:
nit: `syftjson2` seems like a less than ideal alias, could we just use `model`?

Reply:
I thought I got all of these (goland auto refactor). I'll get it back to model.